config/sets/symfony/symfony62.php

```diff
@@ -86,4 +87,6 @@
             'MAX_USERNAME_LENGTH'
         ),
     ]);
+
+    $rectorConfig->rule(ParamConverterAttributeToMapEntityAttributeRector::class);
```

Comment:
@samsonasik there was some sort of guideline about setting rules was it specific rules like these should be done last? If so should we also move `SimplifyFormRenderingRector`  arround here?